relevanceai/dataset/read/metadata.py

```diff
@@ -16,28 +16,31 @@ def __repr__(self):
     def __contains__(self, m):
         return m in self.to_dict().keys()
 
-    def insert_metadata(self, metadata: dict, verbose: bool = False):
+    def insert(self, metadata: dict, verbose: bool = False):
```

Comment:
This wont break anything right?

Reply:
Unless another client project uses the old terminology, it shouldnt